### PR TITLE
Fixed formatting and spacing issues

### PR DESCRIPTION
**Pull request by rogrange**
Fixed several link formatting issues.. Cleaned up some paragraph structures by deleting forced returns, and ensured several stray formatting marks were removed.

### CODE REVIEW

**Review by emurtoug — APPROVED** [review]

File: downstream/modules/platform/proc-accessing-rpm-repositories-for-locally-mounted-dvd.adoc [hunk 1]
```diff
@@ -3,8 +3,7 @@
 = Accessing RPM Repositories for Locally Mounted DVD
 
 
-If you re going to access the repositories from the DVD, it is necessary to set
-up a local repository. This section shows how to do that.
+If you re going to access the repositories from the DVD, it is necessary to set up a local repository. This section shows how to do that.
```
[the remark this answers is not in the document]
Comment (rogrange) [reply downstream/modules/platform/proc-accessing-rpm-repositories-for-locally-mounted-dvd.adoc:6]: Done.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 2]
```diff
@@ -40,8 +38,7 @@ $ sudo -i
 
 === Copying and Pasting
 
-If SSH is unavailable as root between the private automation hub and the controller, copy the contents of the file __/etc/pulp/certs/root.crt__ on the private automation hub and paste it into a new file on the controller called
-__/etc/pki/ca-trust/source/anchors/automationhub-root.crt__. After the new file is created, run the command `update-ca-trust` command in order to update the CA trust store with the new certificate.
+If SSH is unavailable as root between the private automation hub and the controller, copy the contents of the file __/etc/pulp/certs/root.crt__ on the private automation hub and paste it into a new file on the controller called __/etc/pki/ca-trust/source/anchors/automationhub-root.crt__. After the new file is created, run the command `update-ca-trust` command in order to update the CA trust store with the new certificate.
```
[the remark this answers is not in the document]
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc:41]: Done.

File: downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc [hunk 3]
```diff
@@ -22,21 +22,17 @@ To perform a reposync you need a RHEL host that has access to the Internet. Afte
...
 
-.. If not using `--newest-only` the repos downloaded will be ~90GB
+.. If not using `--newest-only` the repos downloaded will be ~90GB.
 
-.. If using `--newest-only` the repos downloaded will be ~14GB
+.. If using `--newest-only` the repos downloaded will be ~14GB.
 
-. If you plan to use Red Hat Single Sign-On (RHSSO)  you must also sync these
-repositories
+. If you plan to use Red Hat Single Sign-On (RHSSO)  you must also sync these repositories.
```
[the remark this answers is not in the document]
Comment (rogrange) [reply downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc:31]: Done

File: downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc [hunk 4]
```diff
@@ -10,31 +10,24 @@ Install Ansible Automation Platform (AAP) automation controller and a private au
...
-servers
+To install AAP on a disconnected network, complete the following prerequisites:
 
-NOTE: The setup bundle includes additional components such as the AAP RPMs, and the default execution environment (EE) images which makes installing AAP easier in a disconnected environment.  The setup bundle tarball is preferred over the
-standard setup tarball for disconnected networks.
+* Create a subscription manifest.
+* Download the AAP setup bundle.
+* Create DNS records for the automation controller and private automation hub servers.
+
+NOTE: The setup bundle includes additional components such as the AAP RPMs, and the default execution environment (EE) images which makes installing AAP easier in a disconnected environment. The setup bundle tarball is preferred over the standard setup tarball for disconnected networks.
```
[the remark this answers is not in the document]
Comment (rogrange) [reply downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc:19]: Good catch. It does make it easier to read.

**Review by emurtoug — APPROVED** [review]
LGTM!

**Pull request merged**